app.py

```diff
@@ -38,18 +39,20 @@ def register():
 
 
 @app.route('/')
+@app.route('/listing')
```

Comment:
Is this the proper way to do this in Flask? Wouldn’t a redirection from `/` to `/listing` be better?